sqlest/src/main/scala/sqlest/ast/Relation.scala

```diff
@@ -155,6 +155,8 @@ case class Select[A, R <: Relation](
 
   def columns = aliasedColumns.columnList(cols)
 
+  def transform[S <: sqlest.ast.Relation](f: sqlest.ast.Select[A, R] => sqlest.ast.Select[A, S]): sqlest.ast.Select[A, S] = f(this)
```

Comment:
You don't need to prefix these types with `sqlest.ast` here (this class is already in that package)